src/sdl2/pixels.rs

```diff
@@ -355,6 +364,8 @@ impl PixelFormatEnum {
             PixelFormatEnum::YUY2 | PixelFormatEnum::UYVY | PixelFormatEnum::YVYU => {
                 num_of_pixels * 2
             }
+            // FIXME: rounding error here?
```

Comment:
What does this FIXME mean? Did you need to check something but forgot?

Reply:
Just copied it from YV12, IYUV formats:
https://github.com/Rust-SDL2/rust-sdl2/blob/5f8a64f6c3d1505e1d252e65c2a7eed7666db2eb/src/sdl2/pixels.rs#L362

Though YV12, IYUV formats have additional size constraints, that prevent rounding error there.
https://github.com/Rust-SDL2/rust-sdl2/blob/master/src/sdl2/render.rs#L1968
https://github.com/Rust-SDL2/rust-sdl2/blob/master/src/sdl2/render.rs#L815
I'll add these to NV12, NV21 as well.